contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -151,18 +153,105 @@ VueJs is a relatively new JavaScript framework as compared to Angular and React.
 2. **Language barriers:**<br/>
 A majority of users of VueJs are the Chinese as VueJs is developed by a Chinese American. He is supportive of the Chinese community and hence a lot of the existing plugins are written in Chinese. There might be some language barriers for an English speaking developer seeking for VueJs resources.
 
-#### Detailed Comparison of VueJs with other JavaScript frameworks can be found from:
+-----
+### Special features of VueJs
+Here are some features that VueJs differ from the other frameworks used:
```

Comment:
The comparison could have been more obvious, after all, it may seem like different ways of writting it in vue? Maybe adding comment like `//react way` or something?